data/arifaisal123.json

```
@@ -1,21 +1,55 @@
 {
   "name": "Arif Faisal",
-  "bio": "Full Stack Developer",
+  "bio": "I am a Full Stack Developer. Currently learning Django.",
+  "tags": ["Python", "JavaScript", "Sass", "Bootstrap", "Flask"],
+  "socials": [
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/faisalcodes123"
+    },
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/arifaisal123"
+    },
+    {
+      "icon": "FaYoutube",
+      "url": "https://youtube.com/@faisalcodes123"
+    },
+    {
+      "icon": "FaLink",
+      "url": "https://arifaisal123.github.io"
+    },
```

```suggestion
    }
```
sorry, missed this one - extra comma here can be removed as well.

---

thank you, suggested change commited.

---

awesome! merging now!